opensafely/info.py

```diff
@@ -38,15 +37,15 @@ def main():
                 cpu=info["NCPU"],
             )
         )
-    except Exception:
-        sys.exit("Error retreiving docker information")
+    except Exception as exc:
+        raise Exception(f"Error retreiving docker information: {exc}")
```

Comment:
I guess there's nothing strictly wrong with it but raising `Exception` looks weird to my eyes because I only every see it used as a base class. I tend to use `RuntimeError` as the most generic "a thing went wrong" exception if I'm raising something.